vliz-be-opsci/.htaccess

```diff
@@ -0,0 +1,31 @@
+RewriteEngine on
+
+# Add types for .ttl and .jsonld and .json
+AddType text/turtle .ttl
+AddType application/ld+json .jsonld
+AddType application/json .json
+
+# vliz-be-opsci home page
+# https://w3id.org/vliz-be-opsci redirects to https://open-science.vliz.be/
+RewriteRule ^$ https://vliz-be-opsci.github.io/ [R=302,L]
+
+# Ontologies / Vocabularies
+# https://w3id.org/vliz-be-opsci/ns redirects to https://open-science.vliz.be/ns
+RewriteRule ^ns$ https://vliz-be-opsci.github.io/ns/ [R=302,L]
+
+# Add rules for text/turtle files in /ns
+RewriteCond %{HTTP_ACCEPT} text/turtle
+RewriteRule ^ns/(.*)$ https://vliz-be-opsci.github.io/ns/$1.ttl [R=302,L]
```

Comment:
you may want to change this to a 303 redirection

Reply:
Same as for other content negotiation methods. See https://www.w3.org/2001/sw/BestPractices/VM/http-examples/2006-01-18/